docs/cloud/rahti2/images/creating.md

```diff
@@ -175,6 +175,120 @@ It is also possible to create a build from a given `Dockerfile`:
 ```
 
+### Import from Git (Private Repositories) using the Web GUI
+
+Deploying a private Git repository to Rahti involves setting up the necessary authentication to access your private repository. Without proper authentication, you will see the error "URL is valid but cannot be reached" (seen in the pictures below). Here's how to resolve this using two authentication methods:
+
+#### Option 1: Using a Token for Git Authentication
+
+1. **Generate a Personal Access Token:**
+
+   - **GitHub:**
+     - Go to your GitHub account settings.
+     - Navigate to "Developer settings" > "Personal access tokens".
+     - Click on "Generate new token".
+     - Select the scopes you need (typically, you'll need `repo` scope for private repositories).
+     - Generate the token and copy it.
+   
+   - **GitLab:**
+     - Go to your GitLab profile settings.
+     - Navigate to "Access Tokens".
+     - Give your token a name, select the required scopes (e.g., `api`, `read_repository`), and create the token.
+     - Copy the token.
+
+2. **Add the Token to Rahti:**
+     - Under "Source Secret" choose "Create new Secret"
+     - Name the secret, under "Authentication type" choose "Basic Authentication"
+     - Paste the token and create
+
+#### Option 2: Using a Private SSH Key for Git Authentication
+
+1. **Generate an SSH Key Pair (if you don't have one already):**
+
+   - Open a terminal and run the following command to generate a new SSH key pair:
+     ```sh
+     ssh-keygen -t rsa -b 4096 -C "your_email@example.com"
+     ```
+   - This will create two files: a private key (`id_rsa`) and a public key (`id_rsa.pub`).
+
+2. **Add Your Public Key to Your Git Hosting Service:**
+
+   - **GitHub:**
+     - Go to your GitHub account settings.
+     - Navigate to "SSH and GPG keys".
+     - Click "New SSH key" and paste the contents of your `id_rsa.pub` file.
+   
+   - **GitLab:**
+     - Go to your GitLab profile settings.
+     - Navigate to "SSH Keys".
+     - Add a new SSH key and paste the contents of your `id_rsa.pub` file.
+
+3. **Add the Private SSH Key to Rahti:**
+     - Under "Source Secret" choose "Create new Secret"
+     - Name the secret, under "Authentication type" choose "SSH Key"
+     - Paste the contents of your private SSH key (`id_rsa`) and create
+
+![025a0cf1-2bdb-477e-bce0-9fe77f43b432](https://github.com/user-attachments/assets/2ea0dac4-de5d-4d6a-8853-9ffcbc3dcdc0)
+
+![fd4c203a-f8e3-4994-b1a4-9a2b4960d4be](https://github.com/user-attachments/assets/ea238069-de97-470f-bb86-03980456eb3a)
+
+![434bf283-c7f5-49d1-8f58-72480f084ce2](https://github.com/user-attachments/assets/1c17a62b-f3b6-404d-8568-24e05e00c1c4)
+
+### Import from Git (Private Repositories) using the CLI
+
+This assumes that the users has generated SSH keys and registered their public key with GitHub.
+
+ 
+**Log into OpenShift CLI (`oc`)**:
+   ```bash
+   oc login <cluster-url>
+   ```
+
+**Create a New Project**:
+   ```bash
+   oc new-project <project-name>
```

Comment:
This does not work, it needs csc_project number. Add link to docs.csc.fi where it is explained how to create a new project